tests/e2e/SymfonyFlex/.env

```diff
@@ -0,0 +1,14 @@
+# In all environments, the following files are loaded if they exist,
```

Comment:
For some reasons this SymfonyFlex thing keeps poping up on my machine... I would like to get it fixed in a separate PR